JuiceMaker/JuiceMaker/Controller/ModifyingInventoryViewController.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  ModifyingInventoryViewController.swift
+//  JuiceMaker
+//
+//  Created by 서수영 on 2022/09/06.
+//
+
+import UIKit
+
+class ModifyingInventoryViewController: UIViewController {
+    static let indentify: String = "ModifyingInventoryViewController"
```

Comment:
indentify는 동사형이여서 명사형으로 변경해주는건 어떨까요?

```suggestion
    static let identifier: String = "ModifyingInventoryViewController"
```

Reply:
수정하였습니다!!!